GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
```

Comment:
Do we want to add a date to this? Maybe an initial accepted version date and then an updated date for any changes down the road?

Reply:
could be, but as this document lives in git in is not that much use of the date here

Reply:
agree that a changelog / version history would be useful, will add that at the bottom.